kin-sdk/AccountInfoRetriver.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using kin_base;
+using kin_base.responses;
+
+namespace kin_sdk
+{
+    class AccountInfoRetriver
+    {
+        private static readonly Asset KinAsset = new AssetTypeNative();
+        private readonly Server server;
+        internal AccountInfoRetriver(Server server)
+        {
+            this.server = server;
+        }
+        internal async Task<decimal> GetBalance(string publicAddress)
+        {
+            try
+            {
+                AccountResponse accountResponse = await this.server.Accounts.Account(publicAddress);
+                foreach (Balance balance in accountResponse.Balances)
+                {
+                    if (balance.Asset.Equals(KinAsset))
```

Comment:
If an account exists it should always have the native asset, correct?

Reply:
yes

Reply:
Then there's no reason for   if (balance.Asset.Equals(KinAsset))

Reply:
An account can still have multiple balances (kin + assets), so I need to iterate over them and verify that I return the kin balance